pkg/process/util/api/payload.go

```diff
@@ -8,10 +8,9 @@ package api
 import (
 	"fmt"
 
-	model "github.com/DataDog/agent-payload/v5/process"
-	"github.com/DataDog/zstd_0"
 	"github.com/gogo/protobuf/proto"
 
+	model "github.com/DataDog/agent-payload/v5/process"
```

Comment:
nit: This should still be at the second import block since it's external to the datadog-agent repo.